### PR TITLE
Module to install yara with r2 module inside

### PR DESCRIPTION
**Pull request by plutec**
r2pm -i r2yara
or
sudo r2pm -gi r2yara

### CODE REVIEW

File: db/r2yara [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+R2PM_GIT "https://github.com/Yara-Rules/r2yara"
+R2PM_DESC "[syspkg] r2 module for yara. Powerful of r2 functionalities in Yara"
+
+R2PM_INSTALL() {
+    # Yara
+    git clone https://github.com/VirusTotal/yara
+
+    # R2 module for Yara
+    cp r2.c yara/libyara/modules/r2.c
```
Comment (radare) [inline db/r2yara]: Add -f

File: db/r2yara [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+    tar -xzvf jansson-2.7.tar.gz
+    cd jansson-2.7
+    ./configure --prefix="${R2PM_PREFIX}"
+    ${MAKE}
+    ${R2PM_SUDO} ${MAKE} install
+    cd ..
+
+    # Compile Yara
+    cd yara
+    export EXT_SO=$(r2 -hh|grep LIBEXT|awk '{print $2}')
```
Comment (radare) [inline db/r2yara]: Isnt that exposed by r2pm already?

File: db/r2yara [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+    wget http://www.digip.org/jansson/releases/jansson-2.7.tar.gz
+    tar -xzvf jansson-2.7.tar.gz
+    cd jansson-2.7
+    ./configure --prefix="${R2PM_PREFIX}"
+    ${MAKE}
+    ${R2PM_SUDO} ${MAKE} install
+    cd ..
+
+    # Compile Yara
+    cd yara
```
Comment (radare) [inline db/r2yara]: Add || error chk here

File: db/r2yara [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+    echo "AM_CFLAGS += @DOLLAR_SIGN@(shell pkg-config --cflags r_socket)" >> yara/Makefile.am
+    echo "LIBS += @DOLLAR_SIGN@(shell pkg-config --libs r_socket)" >> yara/Makefile.am
+
+    # libjansson
+    wget http://www.digip.org/jansson/releases/jansson-2.7.tar.gz
+    tar -xzvf jansson-2.7.tar.gz
+    cd jansson-2.7
+    ./configure --prefix="${R2PM_PREFIX}"
+    ${MAKE}
+    ${R2PM_SUDO} ${MAKE} install
```
Comment (radare) [inline db/r2yara]: Check error here

File: db/r2yara [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+    cp dist/yara-3.6.3/configure.ac yara/configure.ac
+    echo "AM_CFLAGS += @DOLLAR_SIGN@(shell pkg-config --cflags r_socket)" >> yara/Makefile.am
+    echo "LIBS += @DOLLAR_SIGN@(shell pkg-config --libs r_socket)" >> yara/Makefile.am
+
+    # libjansson
+    wget http://www.digip.org/jansson/releases/jansson-2.7.tar.gz
+    tar -xzvf jansson-2.7.tar.gz
+    cd jansson-2.7
+    ./configure --prefix="${R2PM_PREFIX}"
+    ${MAKE}
```
Comment (radare) [inline db/r2yara]: Chk err here

File: db/r2yara [hunk 6]
```diff
@@ -0,0 +1,40 @@
+R2PM_BEGIN
+
+R2PM_GIT "https://github.com/Yara-Rules/r2yara"
+R2PM_DESC "[syspkg] r2 module for yara. Powerful of r2 functionalities in Yara"
+
+R2PM_INSTALL() {
+    # Yara
+    git clone https://github.com/VirusTotal/yara
```
Comment (radare) [inline db/r2yara]: Add error chk

File: db/r2yara (hunk 2 above)
Comment (plutec) [reply db/r2yara]: I removed it, that's true

File: db/r2yara [hunk 7]
```diff
@@ -0,0 +1,39 @@
+R2PM_BEGIN
+
+R2PM_GIT "https://github.com/Yara-Rules/r2yara"
+R2PM_DESC "[syspkg] r2 module for yara. Powerful of r2 functionalities in Yara"
+
+R2PM_INSTALL() {
+    # Yara
+    git clone https://github.com/VirusTotal/yara || R2PM_FAIL "Cloning Yara"
```
Comment (radare) [inline db/r2yara]: Those seems like spaces. We use tabs :)

File: db/r2yara [hunk 8]
```diff
@@ -0,0 +1,39 @@
...
+    cp -f dist/yara-3.6.3/libyara/modules/module_list yara/libyara/modules/module_list
+    cp -f dist/yara-3.6.3/libyara/Makefile.am yara/libyara/Makefile.am
+    cp -f dist/yara-3.6.3/configure.ac yara/configure.ac
+    echo "AM_CFLAGS += @DOLLAR_SIGN@(shell pkg-config --cflags r_socket)" >> yara/Makefile.am
+    echo "LIBS += @DOLLAR_SIGN@(shell pkg-config --libs r_socket)" >> yara/Makefile.am
+
+    # libjansson
+    wget http://www.digip.org/jansson/releases/jansson-2.7.tar.gz
+    tar -xzvf jansson-2.7.tar.gz
+    cd jansson-2.7
```
Comment (radare) [inline db/r2yara]: Missing error chk

File: db/r2yara (hunk 7 above)
Comment (plutec) [reply db/r2yara]: Haha, ok ok, I'm going to change it

**Pull request merged**